packages/TelegramClient-UI.package/TCUChatWindow.class/instance/createTitleBar.st

```diff
@@ -6,9 +6,11 @@ createTitleBar
 		color: TCUDefaultValues colorLightGray;
 		width: self width;
 		height: self defaultTitleBarHeigth;
-		topLeft: (self topLeft);
+		topLeft: self topLeft;
 		layoutPolicy: TableLayout new;
 		listDirection: #leftToRight;
 		layoutInset: 2;
 		vResizing: #shrinkWrap;
-		cellInset: 12).
\ No newline at end of file
+		cellInset: 12).
+	
+	self titleBar on: #mouseUp send: #showInfoPage to: self.
```

Comment:
Is there a reason for using "mouseUp" here? From a little bit of testig, Squeak does not do what would be expected from using this signal. (If you click the group title, drag away and then release the mouse, you still activate the info-page)

Reply:
No, not really any good reason. I think we saw mouseUp being used at some other place of the telegram UI but you're right, mouseDown is better. Changed it now